packages/sakuli-legacy/src/context/common/test-case/test-case.class.spec.ts

```diff
@@ -12,6 +12,7 @@ import {TestStepCache} from "./steps-cache/test-step-cache.class";
 import {TestStep} from "./__mocks__/test-step.function";
 import {LegacyProjectProperties} from "../../../loader/legacy-project-properties.class";
 import {tmpdir} from "os";
+import { exportNamedDeclaration } from "@babel/types";
```

Comment:
I can't find any uses of this import? Is it really required?

Reply:
No this was somehow imported when I on a wrong autocompletion :)